### PR TITLE
Initial class web page update for Fall 2023

### PR DESCRIPTION
**Pull request by sschang17**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -4,23 +4,20 @@ url: https://cs291.com # the base hostname & protocol for your site
...
 
-instructor: Dr Nevena Golubovic
-instructor_email: nevena@cs.ucsb.edu
-instructor_url: https://www.linkedin.com/in/nevenagolubovic/
+instructor: Dr. Shyr-Shea Chang
+instructor_email: shyr-shea.chang@appfolio.com
+instructor_url: https://www.linkedin.com/in/shyr-shea-chang-6426271b8/
 
 drive_url: https://drive.google.com/drive/folders/11ZU37_a1Yc6pTBNo9uIcsCKV422Ypdyf?usp=sharing
 piazza_url: https://piazza.com/ucsb/fall2022/cs291a
```
Comment (nevenag) [inline _config.yml:17]: We need to update those too. Create a new piazza class and update the link here. Same for Google drive (making sure we have access to it)

File: schedule.md [hunk 2]
```diff
@@ -53,16 +53,14 @@ The following schedule is subject to change, and many slide links are not yet ac
 </div>
 <div class="week">
 
-## Week 2 (October 4-6)
+## Week 2 (October 9-11)
 
 ### Topics
 
 - Scaling Web Applications ([slides](/slides/2022/05_scaling_web_applications/))
```
Comment (nevenag) [inline schedule.md:60]: Probably want to drop all the links to slides and update them once the class/content for that week is up.

File: schedule.md [hunk 3]
```diff
@@ -111,21 +110,22 @@ The following schedule is subject to change, and many slide links are not yet ac
...
-## Week 6 (November 1-3)
+## Week 6 (November 6-8)
 
 ### Topics
 
-### Thursday November 3: Guest Lecture with David Acevedo
+### Wednesday November 8: Guest Lecture with David Acevedo
 
 - Deep Dive: Load Testing with Tsung ([slides](/slides/2022/10_tsung))
 - Visiting [North Hall Data Center](https://www.it.ucsb.edu/enterprise-technology-services/north-hall-data-center)
```
Comment (nevenag) [inline schedule.md:131]: This will need to be arranged ahead of time if you want to organize a visit. Let's DM on this.

**Review by nevenag — APPROVED** [review]
LGTM. Few comments on what can be updated.

File: _config.yml (hunk 1 above)
Comment (sschang17) [reply _config.yml]: Done!

**Pull request merged**